nilearn/maskers/_mixin.py

```diff
@@ -26,24 +25,6 @@
 class _MultiMixin:
     """Mixin class to add common MultiMasker functionalities."""
 
-    def __sklearn_tags__(self):
```

Comment:
testing to see if things now just work fine with fewer of those __sklearn_tags__ methods